File: app/src/main/java/com/friday/ar/plugin/file/ZippedPluginFile.java
package com.friday.ar.plugin.file;

import net.lingala.zip4j.core.ZipFile;
import net.lingala.zip4j.exception.ZipException;

import java.io.File;

public class ZippedPluginFile extends ZipFile {
    public ZippedPluginFile(String zipFile) throws ZipException {
        super(zipFile);
    }

    public ZippedPluginFile(File zipFile) throws ZipException {
        super(zipFile);
    }
}

File: app/src/main/java/com/friday/ar/plugin/file/ZippedPluginFile.kt
package com.friday.ar.plugin.file


import net.lingala.zip4j.core.ZipFile
import net.lingala.zip4j.exception.ZipException


import java.io.File

class ZippedPluginFile @Throws(ZipException::class)
constructor(zipFile: File) : ZipFile(zipFile)
